src/main/java/Duke.java

```diff
@@ -1,10 +1,424 @@
-    }
-}
+
+    public static final String SEPARATING_LINE = "  ________________________________________";
+    public static final String LINE_DOWN = "|";
+    public static final int TODO_LEN = 5;
+    public static final int EVENT_LEN = 6;
+    public static final int DEADLINE_LEN = 9;
+    public static final int DONE_LEN = 5;
+<<<<<<< HEAD
+    public static final int DELETE_LEN = 7;
+=======
+    public static final String TASK_FILE = "data/tasks.txt";
+    public static final String TASK_FOLDER = "data";
+>>>>>>> branch-Level-7
+    static ArrayList<Task> tasksToDo = new ArrayList<>();
+
+    public static void main(String[] args){
+        greet();
+        GetCommand();
+        bye();
+    }
+    
+    /**
+    * Greets the user 
+    */
+    private static void greet(){
+        System.out.println(SEPARATING_LINE);
+        System.out.println("Welcome back, master");
+        System.out.println("Your command is my command...");
+        System.out.println(SEPARATING_LINE);
+        System.out.println();
+    }
+
+    /**
+    * Continuously gets command from the user
+    * until the user types out "bye".
+    * Also prompts the user to put in valid commands
+    * by resolving exceptions
+    */
+    private static void GetCommand(){
+        Scanner input = new Scanner(System.in);
+
+        try{
+            readTaskFile();
+            // If there are existing tasks then he is an 
+            // existing user
+            if(tasksToDo.size() > 0){
+                System.out.println("Here is the list that you left off with: ");
+                listOutTasks();
+            } 
+        } catch(IOException e){
+            System.out.println("Sorry master, I am having difficulties finding" + 
+                                " the tasks you left off with");
+        }
+
+        while(true){
+            System.out.println("Please give me your command");
+
+            String command;
+
+            try{
+                command = input.nextLine();
+
+                // If user did not put any command, then 
+                // throw empty command exception
+                if(command == null || command.isEmpty()){
+                    throw new EmptyCommandException();
+                }
+
+                if(command.equals("bye")) {
+                    return;
+                } else if(command.equals("list")){
+                    listOutTasks();
+                } else if(command.contains("done")){
+                    markAsDone(command);
+                } else if(command.contains("todo")){
+                    addNewToDo(command);
+                } else if(command.contains("deadline")){
+                    addDeadline(command);
+                } else if(command.contains("event")){
+                    addEvent(command);
+                } else if (command.contains("delete")){
+                    deleteTask(command);
+                } else{
+                    throw new InvalidCommandException();
```

Comment:
You may want to extract out the commands as classes to implement an OOP design to your Duke